README.md

```diff
@@ -33,7 +33,7 @@ You may also find packages on the author's personal repository locations version
     $ sudo apt update
     $ sudo apt install ghostwriter
 
-Fedora users can install older version of *ghostwriter* from [Copr](https://copr.fedorainfracloud.org/) by opening a terminal and entering the following commands:
+Fedora users can install an older version of *ghostwriter* from [Copr](https://copr.fedorainfracloud.org/) by opening a terminal and entering the following commands:
```

Comment:
I hate to nitpick, but wouldn't it make more sense to use `install older versions`?

Reply:
Both can work, your suggestion may imply Fedora users can install multiple versions simultaneously.

Reply:
Good point! I retract my previous statement.